### DISCUSSION TITLE
Joper BPR

### DISCUSSION THREAD

**Discussion by mo3adz** [artisan-roaster-scope/artisan#1302] [General]
Greetings all this is my first time using Artisan for roasting and I have few questions about how to run it on my Joper BPR . I understand that in the next release ( plus version) it will be supporting the Joper without installing any phidgets . Please correct me if I'm wrong. 

Your advice is highly appreciated and thank you all in advance for your support .

**Comment by mo3adz** [comment]
![image](https://github.com/artisan-roaster-scope/artisan/assets/151010166/70d03f92-5219-4c53-a35e-5083652aa8ba)

**Reply by MAKOMO** [reply]
There is no plus version of Artisan and there never will be. There is the commercial [artisan.plus](https:\\artisan.plus) service providing online inventory management and is tightly integrated with Artisan.

The next Artisan release will provide a setup to connect to selected roasting machines of Joper featuring a PLC. To find out if your machine will be support please contact the excellent support of Joper directly.
